homeassistant/components/media_player/__init__.py

```diff
@@ -54,6 +56,8 @@
 ATTR_APP_ID = 'app_id'
 ATTR_APP_NAME = 'app_name'
 ATTR_SUPPORTED_MEDIA_COMMANDS = 'supported_media_commands'
+ATTR_CURRENT_SOURCE = 'current_source'
+ATTR_SELECT_SOURCE = 'source'
```

Comment:
Can't these two attributes, 'current_source' and 'source' be merged into one 'source' attribute? Then use the same attribute in the service as for the property/state attribute.